### PR TITLE
fix(ios): fix ConsoleModule.log not working

### PR DESCRIPTION
**Pull request by wwwcg**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

File: examples/hippy-react-demo/src/app.jsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ export default class App extends Component {
   }
 
   render() {
```
Comment (zoomchan-cxj) [inline examples/hippy-react-demo/src/app.jsx:46]: 不要在render里加这个逻辑，要么就在初始化的时候

**Review by zoomchan-cxj — CHANGES_REQUESTED** [review]

File: examples/hippy-react-demo/src/app.jsx (hunk 1 above)
Comment (wwwcg) [reply examples/hippy-react-demo/src/app.jsx]: 已调整

**Review by ozonelmy — APPROVED** [review]

File: examples/hippy-react-demo/src/app.jsx [hunk 2]
```diff
@@ -42,6 +43,10 @@ export default class App extends Component {
     });
   }
```
Comment (churchill-zhang) [inline examples/hippy-react-demo/src/app.jsx:45]: 调试代码记得删了

File: ios/sdk/utils/HippyLog.mm [hunk 3]
```diff
@@ -60,39 +59,17 @@ static void registerTDFLogHandler() {
...
                 if (string.length()) {
                     NSString *message = [NSString stringWithUTF8String:string.c_str()];
-                    NSString *fileName = nil;
-                    int lineNumber = 0;
-                    if (getFileNameAndLineNumberFromLogMessage(message, &fileName, &lineNumber)) {
-                        _HippyLogNativeInternal(HippyLogLevelInfo, [fileName UTF8String], lineNumber, @"%@", message);
+                    if (message == nil) { // fixme: deal with unicode characters
+                        message = [NSString stringWithCString:string.c_str() encoding:NSASCIIStringEncoding];
                     }
+                    _HippyLogNativeInternal(HippyLogLevelInfo, "TDFCore", 0, @"%@", message);
```
Comment (churchill-zhang) [inline ios/sdk/utils/HippyLog.mm:65]: 这里确定是TDFCore吗

**Review by churchill-zhang — DISMISSED** [review]

**Review by zoomchan-cxj — DISMISSED** [review]
LGTM

**Review by ozonelmy — APPROVED** [review]

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Review by churchill-zhang — APPROVED** [review]

**Pull request merged**